src/database/schema/swaps_v2.ts

```diff
@@ -0,0 +1,51 @@
+import { relations } from 'drizzle-orm';
+import { pgTable, serial, varchar, integer, timestamp, unique, jsonb, char } from 'drizzle-orm/pg-core';
+
+import { SwapExtra } from 'typings/subgraph/liquidity';
+
+import { chains } from './chains';
+import { poolsTable } from './pools';
+import { tokens } from './tokens';
+
+export const swapsTableV2 = pgTable(
+  'swaps_v2',
+  {
+    id: serial('id').primaryKey(),
+    chainId: integer('chain_id')
+      .notNull()
+      .references(() => chains.id, { onDelete: 'cascade' }),
+    transactionHash: char('transaction_hash', { length: 66 }).notNull(),
+    baseAmount: varchar('base_amount', { length: 256 }).notNull().default('0'),
+    quoteAmount: varchar('quote_amount', { length: 256 }).notNull().default('0'),
+    fees: varchar('fees', { length: 256 }).default('0'),
```

Comment:
add price to schema too:

```suggestion
    price: varchar('price', { length: 256 }).default('0'),
```